caenbrew/packages/nodejs.py

```diff
@@ -0,0 +1,13 @@
+from ..packaging import package, AutotoolsPackage
```

Comment:
I think you're going to be yelled at for not putting these in alphabetical order?

You can `pip install -r requirements-dev.txt` then run `flake8 caenbrew` to catch flakey issues early.